### PR TITLE
updated v2.0.3 migrating keys to include a section on switching clients

### LINKED ISSUE #224: Guide: Migrating Keys Between Eth2 Clients or Between Machines
This is a high priority item for us before we go to mainnet. We need to have detailed documentation regarding migrating validator keys between eth2 clients safely. Although different eth2 clients have various ways of representing keys, we all follow the common EIP-2335 standard for keystore.json files to store secrets. Migrating keys is not an easy feat, as you also need to migrate [`slashing protection histories`](https://hackmd.io/@sproul/Bk0Y0qdGD) which is still a work in progress. We should aim to include the following in this documentation:
- A preface on how Prysm handles validator keys
- How to backup your keys and delete them from Prysm
- How to import keys from an external source, such as from a backup generated by lighthouse's eth2 client
- How to import keys in other eth2 clients (might be tough as if they change their code, our docs become stale)
- How to export your validating history for slashing protection purposes (work in progress)

### PR DESCRIPTION
**Pull request by ib1gymnast**
This PR adds a section on switching clients in the migrating keys portion of the documentation. Specifically, it adds the following steps:

1. Syncing the beacon node
2. Stopping and Disabling Prysm
3. Exporting Slashing Protection History
4. Updating port forwarding rules
5. Importing Validator keys into new validator client
6. Importing Slashing Protection history into new client
7. Starting the new validator. 

Resolves #224

### CODE REVIEW

File: website/docs/advanced/migrating-keys.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: Migrating validator keys between computers
 sidebar_label: Migrating between computers
 ---
 
+Test. 
+
```
Comment (rauljordan) [inline website/docs/advanced/migrating-keys.md:8]:
```suggestion
```

File: website/versioned_docs/version-2.0.3/advanced/migrating-keys.md [hunk 2]
```diff
@@ -348,3 +348,147 @@ Once complete, verify the account removal using the validator accounts list comm
 :::danger Good Practice
 In order to minimise slashing risk, it is recommended that the migrated validator is not started until at least 1 epoch following the last action taken by the original validator. Epoch status can be monitored on multipe websites, however, typically a new epoch is created approximatley every 6.5 minutes, waiting 7.5 minutes after stopping the original validator should significantly reduce the risk of slashing.  
 :::
+
+
+## Switching Clients
```
Comment (rauljordan) [inline website/versioned_docs/version-2.0.3/advanced/migrating-keys.md:353]:
```suggestion
## Guidelines on Switching Between Ethereum Client Software
```

File: website/versioned_docs/version-2.0.3/advanced/migrating-keys.md [hunk 3]
```diff
@@ -348,3 +348,147 @@ Once complete, verify the account removal using the validator accounts list comm
...
+
+This portion of the document will provide guidance on switching clients. These steps will be similar to the above steps in Migrating Computers with some key differences. A portion of this guide will be redundant to areas covered elsewhere in the documentation and will link to those places where it is relevant.
```
Comment (rauljordan) [inline website/versioned_docs/version-2.0.3/advanced/migrating-keys.md:355]:
```suggestion
This portion of the document will provide guidance on switching between Prysm and other consensus clients, such as Teku, Lighthouse, or Nimbus. These steps will be similar to the above steps in Migrating Computers with some key differences. A portion of this guide will be redundant to areas covered elsewhere in the documentation and will link to those places where it is relevant. 
```

File: website/versioned_docs/version-2.0.3/advanced/migrating-keys.md [hunk 4]
```diff
@@ -348,3 +348,147 @@ Once complete, verify the account removal using the validator accounts list comm
 :::danger Good Practice
 In order to minimise slashing risk, it is recommended that the migrated validator is not started until at least 1 epoch following the last action taken by the original validator. Epoch status can be monitored on multipe websites, however, typically a new epoch is created approximatley every 6.5 minutes, waiting 7.5 minutes after stopping the original validator should significantly reduce the risk of slashing.  
 :::
+
+
+## Switching Clients
+
+This portion of the document will provide guidance on switching clients. These steps will be similar to the above steps in Migrating Computers with some key differences. A portion of this guide will be redundant to areas covered elsewhere in the documentation and will link to those places where it is relevant. 
+
+Please see Pryms's note on the importance of client diversity [here.](https://medium.com/prysmatic-labs/prysmatic-labs-statement-on-client-diversity-c0e3c2f05671)
```
Comment (rauljordan) [inline website/versioned_docs/version-2.0.3/advanced/migrating-keys.md:357]:
```suggestion
Regarding why one may wish to switch to a different client, see our team's note on the importance of client diversity [here](https://medium.com/prysmatic-labs/prysmatic-labs-statement-on-client-diversity-c0e3c2f05671).
```

**Comment by rauljordan** [comment]
This is really good! Thank you @ib1gymnast. Just had a few recommendations and then we can merge this in :). I edited your PR description to say it resolves an associated issue on Github.

**Comment by rauljordan** [comment]
Hi @ib1gymnast, hope you are doing well! Could you help us make the same changes you added in this PR for the v2.0.6 versioned docs as well as under the docs/ folder ?

**Comment by ib1gymnast** [comment]
Raul, see my latest PR #412

**Comment by rauljordan** [comment]
Closing in favor of #412

**Pull request closed without merging**